css/src/components/message.scss

```diff
@@ -0,0 +1,52 @@
+.message {
+	display: grid;
+	grid-template:
+		'. time'
+		'avatar main';
+	grid-template-columns: max-content 1fr;
+	gap: $layout-1;
+
+	.message-time {
+		grid-area: time;
+		font-size: $font-size-9;
+	}
+
+	.message-avatar {
+		grid-area: avatar;
+		margin-block-start: $spacer-3;
+	}
+
+	.message-content {
+		grid-area: main;
```

Comment:
Let's scope our grid areas to the component too: `message-main` or `message-content`. `main` is a somewhat loaded term